MicrowaveGame/Assets/Resources/Scripts/Dialogue/DialogueTrigger.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DialogueTrigger : MonoBehaviour
+{
+    ///<summary>
+    ///Attach this script to anything that we want to trigger an explanation (i.e. First look at an enemy)
+    ///</summary>
+
+    /// <summary>
+    /// Dialogue Object
+    /// </summary>
+    public Dialogue dialogue;
```

Comment:
Public fields use `PascalCase` (i.e. `Dialogue`).

Reply:
Would naming the variable the same as the object not break the script (like it wouldn't tell when I'm invoking the object script versus that particular instance?)

Reply:
This would be an issue if you had a field named `Dialogue` in the `Dialogue` class, but since it's inside `DialogueTrigger`, it's okay.